app/build.gradle

```diff
@@ -17,8 +19,12 @@ android {
     }
 
     buildTypes {
-        release {
```

Comment:
Please restore the release build type.
We still want to work with 2 build types (debug/release).
**We want to Add 2 product flavors (dev/prod)**